WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewController.swift

```diff
@@ -1,9 +1,47 @@
 import UIKit
+import Combine
+import OSLog
 
 class CardReaderSettingsViewController: UIViewController {
+    private var viewmodel: CardReaderSettingsViewModel
+    private var previousChildView: CardReaderSettingsActiveChildView = .none
+    private var subscriptions = Set<AnyCancellable>()
+
+    required init?(coder: NSCoder) {
```

Comment:
Are we going to initialize this VC programatically?

Reply:
No. Not at this time. This was leftover from a previous approach. Will remove.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3699/commits/46c1941a4b75b3595b156f57b130316575113c0e